zeppelin-interpreter/src/main/java/org/apache/zeppelin/interpreter/remote/RemoteInterpreterEventClient.java

```diff
@@ -477,15 +477,18 @@ public void onParaInfosReceived(Map<String, String> infos) {
   /**
    * Wait for eventQueue becomes empty
    */
-  public void waitForEventQueueBecomesEmpty() {
+  public void waitForEventQueueBecomesEmpty(long atMost) {
```

Comment:
I think it would be nice if `atMost` indicated what time unit it wants. Rename it to `atMostMilliseconds`?